bspysmg/model/training.py

```diff
@@ -362,7 +387,25 @@ def postprocess(dataloader, model, criterion, amplification, results_dir,
     return torch.sqrt(running_loss)
 
 
-def to_device(inputs, targets):
+def to_device(inputs : torch.Tensor,
```

Comment:
Why not use just a single argument? It is irrelevant if it is inputs and targets?

Reply:
Made necessary changes.